tools/rapids-github-run-id

```diff
@@ -5,13 +5,65 @@
 set -euo pipefail
 export RAPIDS_SCRIPT_NAME="rapids-github-run-id"
 
+function rapids-retry-capture-output {
+  local command=$1
+  shift
+  local max_retries=${RAPIDS_RETRY_MAX:=3}
+  local retries=0
+  local sleep_interval=${RAPIDS_RETRY_SLEEP:=10}
+  local output
+  local retcode
+  
+  output=$("${command}" "$@")
+  retcode=$?
+  
+  while (( retcode != 0 )) && (( retries < max_retries )); do
+    ((retries++))
+    rapids-logger "rapids-retry: retry ${retries} of ${max_retries} | exit code: (${retcode}) -> sleeping for ${sleep_interval} seconds..." >&2
+    sleep "${sleep_interval}"
+    rapids-logger "rapids-retry: sleep done -> retrying..." >&2
+    
+    output=$("${command}" "$@")
+    retcode=$?
+  done
+  
+  if (( retcode == 0 )); then
+    echo "${output}"
+  fi
```

Comment:
```suggestion
  while (( retcode != 0 )) && (( retries < max_retries )); do
    ((retries++))
    sleep "${sleep_interval}"
    output=$("${command}" "$@")
    retcode=$?
  done
  
  if (( retcode != 0 )); then
    echo "ERROR: '${command}' failed on ${max_retries} attempts. If you think more retries would fix this issue, try setting environment variable RAPIDS_RETRY_MAX to a higher value."
    return ${retcode}
  fi
  return 0
```

I think it'd be confusing to see these log lines claiming this is from `rapids-retry`, when it isn't. For simplicity and stronger guarantees that no log output will get into the output, let's just remove these logging lines and move the diagnostic information to the error case.